v0/destinations/active_campaign/data/ACPage.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "destKey": "name",
-    "sourceKeys": ["properties.domain", "properties.url", "context.page.url"],
+    "sourceKeys": ["properties.referring_domain", "context.page.referring_domain", "properties.domain", "properties.url", "context.page.url"],
```

Comment:
Lets do one more thing lets do URL() parsing for extra set of guard rails

Reply:
Why update the precedence order? We should add the new options at the end

Reply:
I had initially thought of giving higher precedence to the `referrer_domain` properties. But we have handled the case of a normal url in the metadata so fixing it.

Reply:
I missed this we should maintain this for backward compatibility

Reply:
I had updated the order.

Reply:
@yashasvibajpai while we are at it can we update the test cases to ga4 standard?

Reply:
New format